.c3i/authorized_users.yml

```diff
@@ -1265,3 +1265,5 @@ authorized_users:
 - pgrossomoreira
 - wu-vincent
 - Inujel
+- Mike-Solar
```

Comment:
```suggestion
```

No answer from Mike-Solar

Reply:
But we'll get them every week no?

Reply:
unfortunately yes. We would need a deny list feature.